pkgs/servers/web-apps/discourse/default.nix

```diff
@@ -444,6 +444,12 @@ let
       maintainers = with maintainers; [ talyz ];
       license = licenses.gpl2Plus;
       description = "Discourse is an open source discussion platform";
+      knownVulnerabilities = [
+        "Various security issues were fixed in newer releases that cannot be backported to NixOS 24.11."
```

Comment:
This should say that it hasn't been maintained and that's why there has been no backport. There could be one, it's just that simply nobody bothered.